python_modules/dagster/dagster_tests/execution_tests/pipes_tests/test_subprocess.py

```diff
@@ -890,13 +890,12 @@ def foo(context: OpExecutionContext, pipes_client: PipesSubprocessClient):
             cmd = [_PYTHON_EXECUTABLE, external_script]
             return pipes_client.run(command=cmd, context=context).get_materialize_result()
 
-    with pytest.warns() as record:
```

Comment:
`pytest.warns()` yells when no warning is detected at all - using [recwarn](https://docs.pytest.org/en/stable/how-to/capture-warnings.html#recwarn) instead.